e2e/tests/integrations/integrator/edit-integration.test.ts

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from "@playwright/test";
+import { createIntegrationAsIntegrator } from "./create-integration.js";
+import { IntegrationTypes } from "../../types.js";
+import { fakerNL_BE as faker } from "@faker-js/faker";
+
+test.use({ storageState: "playwright/.auth/user.json" });
+
+test("As an integrator I can edit an existing integration", async ({
+  page,
+}) => {
+  const { integrationName } = await createIntegrationAsIntegrator(
+    page,
+    IntegrationTypes.SEARCH_API
+  );
+
+  await page.waitForURL(/https?:\/\/[^/]*\/nl\/integraties(\/.*)?/);
+  await expect(page.getByText(integrationName)).toBeVisible();
+
+  // Edit integration
+  await page.getByRole("button", { name: "Instellingen" }).click();
+
+  await page.getByLabel("Integratienaam").fill(faker.word.adjective());
+  await page.getByLabel("Beschrijving").fill(faker.lorem.lines(2));
+
+
+  await page
+    .locator("div")
+    .filter({ hasText: /^Login URLtestomgevingproductieomgeving$/ })
+    .getByRole("textbox")
+    .first()
+    .fill(faker.internet.url());
+
+  await page
```

Comment:
Same for this one

Reply:
I will do it in a seperate PR.
So I can first make to changes in the html and deploy the changes to acc, in that way I can modify the tests locally and see if they succeed.